src/components/select/index.js

```diff
@@ -0,0 +1,3 @@
+export { default as Option } from './option';
```

Comment:
The import for option is broken.
e.g. `import { Select, Option } from "carbon-react/lib/components/select";` does not work.

Reply:
fixed